src/matlab2tikz.m

```diff
@@ -1366,6 +1367,18 @@ case guitypes()
         end
         str = join(m2t, str, '\\[1ex]');
         opts =  opts_add(opts, tikzKeyword, sprintf('{%s}', str));
+    
+        % set color of axis labels
+        color = object.Color;
+        [m2t, col] = getColor(m2t, handle, color, 'patch');
+        if strcmpi(labelKind,'title')
+          % TODO: color titles
```

Comment:
While you are at it. there is the title style = {} option, where we could set the color. That might be nice to